crates/rpc/src/server/api.rs

```diff
@@ -40,6 +40,7 @@ use crate::COMPONENT;
 pub struct RpcService {
     store: StoreRpcClient,
     block_producer: Option<BlockProducerClient>,
+    genesis_commitment: Word,
```

Comment:
I made a PR to your branch to move this to option also

https://github.com/varun-doshi/miden-node/pull/1